examples/xds-sotw-config-server/go.mod

```diff
@@ -13,8 +13,10 @@ require (
 	github.com/envoyproxy/protoc-gen-validate v0.9.1 // indirect
 	github.com/golang/protobuf v1.5.2 // indirect
 	golang.org/x/net v0.4.0 // indirect
-	golang.org/x/sys v0.3.0 // indirect
-	golang.org/x/text v0.5.0 // indirect
+	golang.org/x/sys v0.5.0 // indirect
+	golang.org/x/text v0.7.0 // indirect
 	google.golang.org/genproto v0.0.0-20221118155620-16455021b5e6 // indirect
 	google.golang.org/protobuf v1.28.1 // indirect
 )
+
+replace golang.org/x/net => golang.org/x/net v0.7.0
```

Comment:
why is this a replace

Reply:
The replace is there such that it replaces an indirect reference.
Something that is directly referenced has a reference to golang.org/x/net - so you can either replace all of those items with newer items that then transitively fix the reference to golang.org/x/net or you can just do the replace operation to replace that reference.

It is a powerful feature of the go build environment that can, when used carefully be very useful in multiple ways (and, as with all powerful features, can be abused too)